tasks/install_yum.yml

```diff
@@ -29,9 +29,21 @@
     - "postgresql{{ postgresql_version_terse }}"
     - "postgresql{{ postgresql_version_terse }}-contrib"
 
-- name: PostgreSQL | PGTune
+- name: PostgreSQL | PGTune | yum
   yum:
     name: pgtune
     state: present
   environment: "{{ postgresql_env }}"
   when: postgresql_pgtune
+
+- name: PostgreSQL | Install Python pip installer | yum
+  yum:
+    name: "{{ postgresql_yum_pip_package }}"
```

Comment:
The package name is different on CentOS 6 and 7... so I've turned this into a variable.